### PR TITLE
Create shopify.sh

### PR DESCRIPTION
**Pull request by hughker**
- Custom Deployment Script to Shopify

### CODE REVIEW

**Comment by hughker** [comment]
@mlocher — per your request, here's the Shopify Deployment Script I've been using and refining.

File: deployments/shopify.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# can be found here - https://github.com/hughker/shopify_theme#configuration
+# You can leave off the SHOPIFY_THEME_ID if you'd like, just remember to remove it
+# from the script before attempting to deploy.
+SHOPIFY_API_KEY=""
+SHOPIFY_API_PASSWORD=""
+SHOPIFY_STORE_URL=""
+SHOPIFY_THEME_ID=""
+
+gem install shopify_theme
+theme configure SHOPIFY_API_KEY SHOPIFY_API_PASSWORD SHOPIFY_STORE_URL SHOPIFY_THEME_ID
```
Comment (mlocher) [inline deployments/shopify.sh]: The line should probably read

```
theme configure "${SHOPIFY_API_KEY}" "${SHOPIFY_API_PASSWORD}" "${SHOPIFY_STORE_URL}" "${SHOPIFY_THEME_ID}"
```

though quotes and the braces are optional (I'm just used to using them in most scripts)

Comment (hughker) [inline deployments/shopify.sh]: sure, I can update it, I don't currently use it in Codeship but now will.

**Comment by mlocher** [comment]
@hughker Thanks so much for providing this! Ping me in app for a t-shirt if you want one!

**Pull request merged**